docs/logging/nats-component.md

```diff
@@ -0,0 +1,163 @@
+---
```

Comment:
NATS is way more than just a logging server. It shouldn't be under "logging". It should be "messaging" more likely.

Reply:
/cc @alexwolfmsft